conda.recipe/meta.yaml

```diff
@@ -31,14 +31,14 @@ requirements:
     - versioningit
 
   build:
-    - mantid
+    - mr_reduction
     - matplotlib
     - setuptools
     - qtpy
     - versioningit
 
   run:
-    - mantid=6.12.0
+    - mr_reduction=2.2.0
     - matplotlib
```

Comment:
Stating `matplotlib`, `qtpy`, and `qt` should be unnecessary because of `mr_reduction` requiring `mantidworkbench`. Please remove them